audit-ci.jsonc

```diff
@@ -0,0 +1,7 @@
+{
+	// $schema provides code completion hints to IDEs.
+	"$schema": "https://github.com/IBM/audit-ci/raw/main/docs/schema.json",
+	"moderate": true,
```

Comment:
I'd prefer to use `low: true` to also get informed about all vulnerabilities, also with prio low.

Reply:
That would mean that the job would fail on any found vulnerability. This is the threshold of failing the job. 
Still, regardless of the threshold, a summary with all the findings could be found within the job.

So, the question is more like "**_Do we want to fail the job if any kind of a security vulnerability is found?_**"
If we set the threshold to `low` it would mean that we'd need to maintain (eventually) a bigger list of exceptions as you're probably aware that many of the low severity issues are most likely not to be resolved sooner or later